pkg/controllers/managementuser/networkpolicy/netpol.go

```diff
@@ -277,7 +356,10 @@ func generateDefaultNamespaceNetworkPolicy(aNS *corev1.Namespace, projectID stri
 		ObjectMeta: v1.ObjectMeta{
 			Name:      defaultNamespacePolicyName,
 			Namespace: aNS.Name,
-			Labels:    labels.Set(map[string]string{nslabels.ProjectIDFieldLabel: projectID}),
+			Labels: map[string]string{
+				nslabels.ProjectIDFieldLabel: projectID,
+				creatorLabel:                 creatorNorman,
```

Comment:
are these not getting added again for a few policies?

Reply:
Yea- I saw a few cases during testing where these labels weren't added. This resulted in some policies not being cleaned up when disabling PNI. I'm setting explicitly here so that we ensure all rancher managed nps get cleaned up when disabling PNI

Reply:
👍🏼 could we double check on all the policies we create and add it for them as well? Good call on testing disabling PNI and catching this.

Reply:
Yep, I made sure all the ones we create are getting the label. I'll do another run through and make sure I got all of them